### ISSUE TITLE
Waste paper sensor no longer works with Omrin

### ISSUE THREAD

**Issue by richardtrinks** [pippyn/Home-Assistant-Sensor-Afvalbeheer#153]
On two different systems I have the problem that waste paper from Omrin remains unknown. Has Omrin changed the name?

![Omrin](https://user-images.githubusercontent.com/15097817/95048135-9f044000-06e7-11eb-9c4c-f81e45adb4c9.png)

**Comment by pippyn** [comment]
Did you try the [option to list all possible fractions](https://github.com/pippyn/Home-Assistant-Sensor-Afvalbeheer#print-all-available-waste-fractions)?
Do you have an address for me to test with?

**Comment by richardtrinks** [comment]
Yes, zip code 8935 SH or 8935 LD

**Comment by richardtrinks** [comment]
- platform: afvalbeheer
    wastecollector: Omrin
    resources:
      - sortibak
      - gft
      - papier
      - grofvuil
    postcode: 8935SH
    streetnumber: ##
    upcomingsensor: 0
    dateformat: '%d-%m-%Y'
    dateonly: 0
    dateobject: 0
    name: ''
    nameprefix: 0
    builtinicons: 0
    disableicons: 1
    dutch: 1

**Comment by karajito** [comment]
Just installed the custom component earlier this week and also for me I am not able to retrieve the 'papier' data.

Looking at the error I get (see below) I think it has to do with the fact that the next paper collection date will be after the end of day light saving on October 27 this year. Here is my log error with a time format mismatch:
 
```
Logger: homeassistant
Source: custom_components/afvalbeheer/sensor.py:877
First occurred: 3:03:50 PM (1 occurrences)
Last logged: 3:03:50 PM

Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/config/custom_components/afvalbeheer/sensor.py", line 430, in async_update
    await self.collector.update()
  File "/config/custom_components/afvalbeheer/sensor.py", line 877, in update
    date=datetime.strptime(item['Datum'], '%Y-%m-%dT%H:%M:%S+02:00'),
  File "/usr/local/lib/python3.8/_strptime.py", line 568, in _strptime_datetime
    tt, fraction, gmtoff_fraction = _strptime(data_string, format)
  File "/usr/local/lib/python3.8/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '2020-11-02T00:00:00+01:00' does not match format '%Y-%m-%dT%H:%M:%S+02:00'
```

**Comment by pippyn** [comment]
fixed in v4.6.8
